src/date-range-picker/__tests__/date-range-picker-absolute.test.tsx

```diff
@@ -53,660 +56,1058 @@ afterEach(() => {
+          });
 
-      test('hh format gets padded with 00:00 for mm:ss when provided via input', () => {
-        const onChangeSpy = jest.fn();
-        const { wrapper } = renderDateRangePicker({
-          ...defaultProps,
-          timeInputFormat: 'hh',
-          onChange: event => onChangeSpy(event.detail),
+          wrapper.findTrigger().click();
+          wrapper.findDropdown()!.findStartDateInput()!.setInputValue(expectedStartDateValue);
+          wrapper.findDropdown()!.findEndDateInput()!.setInputValue(expectedEndDateValue);
+
+          if (granularity === 'day') {
+            wrapper.findDropdown()!.findStartTimeInput()!.setInputValue('05');
+            wrapper.findDropdown()!.findEndTimeInput()!.setInputValue('23');
+          } else {
+            expect(wrapper.findDropdown()!.findStartTimeInput()).toBeNull();
+            expect(wrapper.findDropdown()!.findEndTimeInput()).toBeNull();
+          }
+          wrapper.findDropdown()!.findApplyButton().click();
+          expect(onChangeSpy).toHaveBeenCalledWith(
+            expect.objectContaining(
+              granularity === 'day'
+                ? {
+                    value: {
+                      type: 'absolute',
+                      startDate: '2018-05-10T05:00:00+08:45',
+                      endDate: '2018-05-12T23:00:00+08:45',
+                    },
+                  }
+                : {
+                    value: {
+                      type: 'absolute',
+                      startDate: '2018-05',
+                      endDate: '2018-05',
+                    },
+                  }
+            )
+          );
         });
-
-        wrapper.findTrigger().click();
-        wrapper.findDropdown()!.findStartDateInput()!.setInputValue('2018/05/10');
-        wrapper.findDropdown()!.findEndDateInput()!.setInputValue('2018/05/12');
-        wrapper.findDropdown()!.findStartTimeInput()!.setInputValue('05');
-        wrapper.findDropdown()!.findEndTimeInput()!.setInputValue('23');
-        wrapper.findDropdown()!.findApplyButton().click();
-        expect(onChangeSpy).toHaveBeenCalledWith(
-          expect.objectContaining({
-            value: {
-              type: 'absolute',
-              startDate: '2018-05-10T05:00:00+08:45',
-              endDate: '2018-05-12T23:00:00+08:45',
-            },
-          })
-        );
       });
-    });
 
-    describe('time offset', () => {
-      test('creates values in the provided time offset', () => {
-        const onChangeSpy = jest.fn();
-        const { wrapper } = renderDateRangePicker({
-          ...defaultProps,
-          timeOffset: 6.5 * 60,
-          onChange: event => onChangeSpy(event.detail),
-        });
-
-        wrapper.findTrigger().click();
-        wrapper.findDropdown()!.findStartDateInput()!.setInputValue('2018/05/10');
-        wrapper.findDropdown()!.findEndDateInput()!.setInputValue('2018/05/12');
-        wrapper.findDropdown()!.findApplyButton().click();
+      //todo  fix
```

Comment:
What's left to be fixed here?

Reply:
all todos will be addressed and these tests specifically will be working in parity with what happens for the regular calendar